README.md

```diff
@@ -30,6 +30,20 @@ Build and install
 
 Refer to [INSTALL.md](./INSTALL.md)
 
+Testing
+-------
+
+Besides manual testing, which is also encouraged, there are also a number of
+checks defined that are used for CI (e.g. github actions).
+
+These checks can be listed using `nix flake show`. You can, for example, test
+the current state of the repo or test every patch before publishing them.
+
+```shell
```

Comment:
Thanks for reviewing :)

Finally found the reference for langs gh uses here, think `console` would be best since that's an alias that gh and highlight.js recognize.